lib/plaid/api_client.rb

```diff
@@ -23,6 +23,9 @@
 
 module Plaid
   class ApiClient
+    class ApiTimeoutError < ApiError; end
+    class ApiConnectionFailedError < ApiError; end
```

Comment:
Re https://github.com/plaid/plaid-ruby/pull/483#issuecomment-1912870145, I believe it's a namespace issue

```suggestion
    class ApiTimeoutError < Plaid::ApiError; end
    class ApiConnectionFailedError < Plaid::ApiError; end
```

Or may need to be ::Plaid

Reply:
I believe it's a loading issue, which is only present in the tests. The namespace should work without needing to be explicit.

Reply:
I did try changing the namespace as suggested here but now get the error: 
```
/usr/src/app/lib/plaid/api_client.rb:26:in `<class:ApiClient>': uninitialized constant Plaid::ApiError (NameError)
```
If you can figure out the solution that would be great. If not, things have been a little hectic this week (performance review season, family emergency) but I can also ask some people who actually know Ruby to take a look sometime next week.

Reply:
Thanks! I tried this and the tests pass with these changes; I've submitted a fix for upstream. 

I don't have a ton of insight into CI for this repo -- honestly, I don't have access to CircleCI either and typically run the tests locally using our internal makefile, which builds the libraries using the Docker images provided by OpenApiTools and then runs the tests. Anyway, I can pass the request along to the team that does run it!

Reply:
> Thanks! I tried this and the tests pass with these changes; I've submitted a fix for upstream.
> 
> I don't have a ton of insight into CI for this repo -- honestly, I don't have access to CircleCI either and typically run the tests locally using our internal makefile, which builds the libraries using the Docker images provided by OpenApiTools and then runs the tests. Anyway, I can pass the request along to the team that does run it!

Awesome, glad it worked out and will be in the next release!

Thanks for passing along the CI request. At my company (AppFolio) we also use CircleCI and all of our open source repos have publicly available CI.